test/functional/Completion.hs

```diff
@@ -327,11 +330,21 @@ contextTests = testGroup "contexts" [
     ]
 
 shouldContainCompl :: [CompletionItem] -> T.Text -> Assertion
-compls `shouldContainCompl` x  =
-    any ((== x) . (^. label)) compls
-    @? "Should contain completion: " ++ show x
+compls `shouldContainCompl` lbl  =
+    any ((== lbl) . (^. label)) compls
+    @? "Should contain completion: " ++ show lbl
 
 shouldNotContainCompl :: [CompletionItem] -> T.Text -> Assertion
-compls `shouldNotContainCompl` x =
-    all ((/= x) . (^. label)) compls
-    @? "Should not contain completion: " ++ show x
+compls `shouldNotContainCompl` lbl =
+    all ((/= lbl) . (^. label)) compls
+    @? "Should not contain completion: " ++ show lbl
+
+getCopletionByLabel :: T.Text -> [CompletionItem] -> Session CompletionItem
```

Comment:
Is the `Copletion` a typo? `getCompletionByLabel`

Reply:
Good spot, fixed.